frontend/WikiContrib-Frontend/src/components/activity.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { fetchAsynchronous } from './fetch';
 import { commits_by_date } from './../api';
-import { Card, Placeholder, Header } from 'semantic-ui-react';
-
+import { Card, Placeholder, Header, Popup } from 'semantic-ui-react';
+import gerritPlatformIcon from '../img/gerritPlatformIcon.png';
+import phabricatorPlatformIcon from '../img/phabricatorPlatformIcon.png';
```

Comment:
The Phabricator icon was looking better without the circle, like you had it earlier. Right now it appears wider than the Gerrit icon and it is not looking nice.